reframe/frontend/runreport.py

```diff
@@ -171,7 +170,7 @@ def junit_xml_report(json_report):
             'package': 'reframe',
             'tests': str(json_report['session_info']['num_cases']),
             'time': str(json_report['session_info']['time_elapsed']),
-            'timestamp': isotime.isoformat(),
+            'timestamp': json_report['session_info']['time_start'][:-5],
```

Comment:
I guess it doesn't like the timezone? We should put a short comment then explaining this.

Reply:
The error message is:
```
E   AssertionError: Element 'testsuite', attribute 'timestamp': '2021-05-01T12:09:31+0000' is not a valid value of the atomic type 'ISO8601_DATETIME_PATTERN'.
```